pycam/Importers/STLImporter.py

```diff
@@ -115,6 +123,9 @@ def import_model(filename, use_kdtree=True, callback=None, **kwargs):
     p2 = None
     p3 = None
 
+    # Skip the header and count fields of binary stl file
```

Comment:
I think, you should move this into the `if is_binary` branch, or?

At least I would expect that it currently breaks the text-mode parser.